### PR TITLE
Added a dialog to provide cohort name when doing Export.

### PR DESCRIPTION
**Pull request by malathi13**

### CODE REVIEW

File: ui/src/components/ExportFab.js [hunk 1]
```diff
@@ -5,16 +5,30 @@ import "./ExportFab.css";
...
   constructor(props) {
     super(props);
-    this.handleClick = this.handleClick.bind(this);
+    this.state = {open: false};
+    this.handleClick = this.handleClick.bind(this);     
+    this.handleCancel = this.handleCancel.bind(this);    
+    this.handleSave = this.handleSave.bind(this);    
   }
 
+  state = {
```
Comment (melissachang) [inline ui/src/components/ExportFab.js]: I don't think you need this, `state` is initialized in constructor

File: ui/src/components/ExportFab.js [hunk 2]
```diff
@@ -5,16 +5,30 @@ import "./ExportFab.css";
 import CloudUpload from "@material-ui/icons/CloudUpload";
 import Button from "@material-ui/core/Button";
 import Tooltip from "@material-ui/core/Tooltip";
+import TextField from '@material-ui/core/TextField';
```
Comment (melissachang) [inline ui/src/components/ExportFab.js]: I get an error and warning when I run locally with your branch, does it work for you?

File: ui/src/components/ExportFab.js [hunk 3]
```diff
@@ -5,16 +5,25 @@ import "./ExportFab.css";
...
```
Comment (melissachang) [inline ui/src/components/ExportFab.js]: Can be a separate PR -- Can you make the dialog only show up if something is selected?

File: ui/src/components/ExportFab.js [hunk 4]
```diff
@@ -25,10 +34,42 @@ class ExportFab extends React.Component {
           <CloudUpload />
         </Button>
       </Tooltip>
+       <div>
+        <Dialog
```
Comment (melissachang) [inline ui/src/components/ExportFab.js]: Sorry changed my mind, can you add the Cancel button back in? Thanks.

File: ui/src/components/ExportFab.js [hunk 5]
```diff
@@ -5,16 +5,25 @@ import "./ExportFab.css";
...
-    this.handleClick = this.handleClick.bind(this);
+    this.state = {open: false};
+    this.handleClick = this.handleClick.bind(this);     
+    this.handleSave = this.handleSave.bind(this);    
   }
 
   render() {
     return (
+      <div>
       <Tooltip title="Export to Saturn">
```
Comment (melissachang) [inline ui/src/components/ExportFab.js]: Can you fix the formatting? Eg this line should be indented two spaces compared to previous line. Also, this I think this div should end on line 37 instead of line 63.

Comment (malathi13) [reply ui/src/components/ExportFab.js]: There should be an outer <div> enclosing all the tags, I got an error without it.

File: ui/src/components/ExportFab.js (hunk 4 above)
Comment (malathi13) [reply ui/src/components/ExportFab.js]: Done
